rust-app/src/implementation.rs

```diff
@@ -26,6 +26,7 @@ use core::future::Future;
 
 type IotaAddressRaw = [u8; IOTA_ADDRESS_LENGTH];
 
+#[allow(dead_code)]
```

Comment:
Is it really not used anymore?

Reply:
There was a warning for only the first part of it, so `ledger_device_sdk::ecc::ECPublicKey<65, 'E'>`

Reply:
Can you maybe make it an expect so it warns when/if we start using it again?

Reply:
Sure